src/draggable.coffee

```diff
@@ -19,6 +19,7 @@ class Draggable extends DragAndDrop
   disable: ->
     if @enabled
       @$el.off "dragstart", @options.selector, @_dragStart
+      @enabled = false
```

Comment:
I got no idea, but disabling a draggable doesn’t really change anything, so it can’t really create any bugs